solo-chains/runtime/dancelight/Cargo.toml

```diff
@@ -235,12 +242,17 @@ std = [
 	"parity-scale-codec/std",
 	"polkadot-parachain-primitives/std",
 	"primitives/std",
+	"rand/std",
+	"rand/std",
```

Comment:
I see this one is duplicated, not sure if it will bring problems.

Reply:
fixed